iexec-sms-library/src/main/java/com/iexec/sms/api/config/TeeAppConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package com.iexec.sms.api.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class TeeAppConfiguration {
+    private String image;
+    private String fingerprint;
+    private String entrypoint;
+    private long heapSize;  // In GB
```

Comment:
I usually prefer things like `OkHttpClient` (vs `OkHTTPClient`), but in this context, I agree `heapSizeGB` is better to avoid any confusion :+1:!
We have some parts of the middleware where this value is bytes and others where it is GB, this is why I'm wondering about that part (we already had a regression bug fix)

Reply:
OK, please https://github.com/iExecBlockchainComputing/iexec-sms/pull/119/commits/e3f9605e7b592b75f894132dace04a40356f3ff8 & https://github.com/iExecBlockchainComputing/iexec-worker/pull/469/commits/88472e98fe754cc9adfb97d1d572e87fae70ea20.